content/docs/02.installation/14.podman.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Podman Compose
+icon: /docs/icons/podman.svg
```

Comment:
This icon does not exist in the repo. Can you add it?

Reply:
Added!